packages/renderer/src/lib/image/PullImage.svelte

```diff
@@ -31,7 +31,7 @@ $: initTerminal();
 const lineNumberPerId = new Map<string, number>();
 let lineIndex = 0;
 
-let terminalIntialized = false;
+export let terminalIntialized = false;
```

Comment:
looks like we have a typo on terminalInitialized

Reply:
Fix pushed.